### PR TITLE
Adds new robot codex

### PR DESCRIPTION
**Pull request by elgeonmb**
lore drips from my soft lips as I smile, a bit too wide for my face. take this. a gift from me to you

### CODE REVIEW

File: code/modules/lore_codex/robutt_data/bybrand.dm [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	<br><br>\
+	Cyber Solutions prosthetics are only recommended for the truly destitute and dedicated tinkers."
+
+/datum/lore/codex/page/robo_nt
+	name = "NanoTrasen"
+	data = "A fairly unremarkable prosthetic design, NanoTrasen health plans provide these as part of their famous corporate medical insurance. These are also provided to NanoTrasen-produced positronics at the end of their indenture and are considered fairly high quality for entry-level prosthesis. Outside of NanoTrasen facilities, NanoTrasen-brand prosthetics are fairly rare and not often sought out-- NanoTrasen's grey-black prosthetics are a visual sign of association with the company itself.\
+	<br><br>\
+	NanoTrasen prosthetics are a solid choice for any NanoTrasen employee. They're easy to come accross in the event of a workplace accident and comfortable enough that many never feel the need to upgrade."
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/bybrand.dm]: across

File: code/modules/lore_codex/robutt_data/bybrand.dm [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+/datum/lore/codex/page/robo_wt
+	name = "Ward-Takahashi GMB"
+	data = "The manufacturing giant's prosthetic line is functionally quite similar to Nanotrasen, but is directed at the consumer market and cheaper than Nanotrasen's own product. Aesthetically, of course, they are worlds apart, trading Nanotrasen's black gunmetal for rounded white plastic. The design endeavors to avoid the \"uncanny valley\" that plagues humaniform robot designs, by abstracting away the human shape into a series of connected ovals. Artists debate whether or not it succeed, while normal people are more concerned with WT's tendency to accrue dirt and scratches with hard use.\
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/bybrand.dm]: succeeds

File: code/modules/lore_codex/robutt_data/bybrand.dm [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+	The main downside to Vey-Med is its incredibly high pricetag-- easily comparable with that of a large house. Every Vey-Med piece is custom-sculpted for its individual owner, usually based on scans of the owner's previous body or the owner's family. Vey-Med also requires regular maintenance, lest the partially-organic synthskin that coats each piece begin to rot.\
+	Many nouveau riche and lottery winners have splurged on a Vey-Med chassis, only to discover that they couldn't keep it maintained. Such individuals contribute to the large and semi-legal market of second, third, and forth-hand Vey-Med chasses, which are cheaper than a store-bought one, but not by very much. Vey-Medical forbids resale of its prosthetics line, and works hard to shut down illicit sales. Potential buyers of a pre-owned Vey-Medical chassis should avoid deals that seem too good to be true, lest they discover that their new body is a Zeng-Hu with a layer of epoxy.\
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/bybrand.dm]: fourth-hand

File: code/modules/lore_codex/robutt_data/main_robutts.dm [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/datum/lore/codex/category/main_robutts // The top-level categories for the robot guide
+	name = "Index"
+	data = "Temporary death, emancipation, and even basic body migration can be a stressful period in anyone's life. The prosthetics market is complicated, and many corporations view misleading or outright decieving potential clients as \"part of the game\". This Buyer's Guide was compiled in 2542 to serve as an easy reference to the most popular prosthetics available, with concideration paid to every use-case and price point. This is our 12th edition, and much has changed in the field of prosthetics since we first set out. We hope you find the information in this guide helpful."
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/main_robutts.dm]: consideration

File: code/modules/lore_codex/robutt_data/main_robutts.dm [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+	children = list(
+		/datum/lore/codex/category/pros_by_brand,
+		/datum/lore/codex/page/quickstart,
+		/datum/lore/codex/category/other_information,
+		)
+
+/datum/lore/codex/page/quickstart
+	name = "Quickstart Primer"
+	data = "If you have to make a split decision; go for Hephestus if you can afford it, Ward-Takahashi if you can't. If you're worried about being seen as \"human\", Zeng-Hu does just as good as Vey-Med for most purposes and doesn't cost millions of Thaler. Don't buy Cyber Solutions unless your only alternative is brain-in-a-jar.."
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/main_robutts.dm]: Hephaestus

File: code/modules/lore_codex/robutt_data/more.dm [hunk 6]
```diff
@@ -0,0 +1,34 @@
+/datum/lore/codex/category/other_information
+	name = "More Information"
+	data = "Here you'll find articles dsigned to further-aid your decision making process, narrowing down the brands outlined in the main body."
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/more.dm]: designed to aid

File: code/modules/lore_codex/robutt_data/more.dm [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+/datum/lore/codex/page/posiage
+	name = "Positronics and Age"
+	data = "Young positronics are unlikely to have much in the way of expensive belongings, including prosthetics. Most positronics come into the working world outside of a well-defined familial structure and are still paying off the debt from their first body.\
+	Even those who do come into a family mature very quickly, not providing time for relatives to save up for an expensive chassis before they begin adulthood and working life. Because of these logistical concerns, even a positronic who has been in a well-paying job for a few years is unable to afford a high-quality chassis.\
+	<br><br>\
+	Most positronics buy their second chassis at around 10 to 15 years of age, and it is at these age that positronics in more middling chasses come into being. Positronics who want a high-end body must save for decades while scrambling for advancement in a galaxy that is rather bigotted more often than not."
```
Comment (Atermonera) [inline code/modules/lore_codex/robutt_data/more.dm]: chassis, bigoted

File: code/modules/lore_codex/robutt_data/main_robutts.dm (hunk 5 above)
[the remark this answers is not in the document]
Comment (Belsima) [reply code/modules/lore_codex/robutt_data/main_robutts.dm]: hephaestus

[the remark this answers is not in the document]
Comment (elgeonmb) [reply code/modules/lore_codex/robutt_data/main_robutts.dm]: TSC_HEPH

**Comment by Anewbe** [comment]
Tested?

**Comment by elgeonmb** [comment]
yeah

**Pull request merged**